pepper-apis/studybuilder-cli/studies/singular/patches/medical-record-release_v2.conf

```diff
@@ -1127,7 +1206,32 @@
           },
         },
 
-        // Signature of parent or legally authorized representative:
+        // Date
+        {
+          "blockType": "CONTENT",
+          "shownExpr": """user.studies["singular"].forms["MEDICAL_RECORD_RELEASE"].questions["MRR_WHO_ENROLLING_COPY"].answers.hasOption("DEPENDENT")""",
+          "titleTemplate": {
+            "templateType": "HTML",
+            "templateText": "$medical_record_release_date_title",
+            "variables": [
+              {
+                "name": "medical_record_release_date_title",
+                "translations": [
+                  { "language": "en", "text": ${i18n.en.medical_record_release_v2.questions.date.title} },
+                ],
+              },
+            ],
+          },
+          "bodyTemplate": {
+            "templateType": "HTML",
+            "templateText": """
+              <p class="paragraph">$ddp.date("MM / dd / yyyy")</p>
+            """,
+            "variables": [],
+          },
+        }
+
+        // Signature of parent or legally authorized representative: MRR_PARENT_SIGNATURE_NEW
         {
```

Comment:
Looks like a lot of changes, but essentially I added a new field (MRR_SIGNATURE) and did some reordering. Tried to get Date to show up behind name.